presto-main/src/main/java/com/facebook/presto/execution/CreateTableTask.java

```diff
@@ -115,7 +119,23 @@ public String explain(CreateTable statement, List<Expression> parameters)
                 if (columns.containsKey(name)) {
                     throw new SemanticException(DUPLICATE_COLUMN_NAME, column, "Column name '%s' specified more than once", column.getName());
                 }
-                columns.put(name, new ColumnMetadata(name, type, column.getComment().orElse(null), false));
+
+                Map<String, Object> columnProperties;
+                if (!column.getProperties().isEmpty()) {
```

Comment:
What `column.getProperties().isEmpty()` but connector requires a property to be set? Now `ColumnPropertyManager` does not have a chance to throw an exception.